e2e/_suites/ingest-manager/features/agent_endpoint_integration.feature

```diff
@@ -0,0 +1,31 @@
+@agent_endpoint_integration
+Feature: Fleet Mode Agent with Endpoint Integration
+  Scenarios for Agent to deploy Endpoint and sending data to Ingest Manager and Elasticsearch.
+
+@deploy-endpoint-with-agent
+Scenario: Deploy Endpoint with Agent and see host in Security App
+  Given an agent is deployed to Fleet
+  When the "Endpoint" latest package version is installed successfully
+    And a "Endpoint" package datasource is added to the 'default' configuration
+  Then the "default" configuration shows the "Endpoint" datasource added
+    And the host name is shown in the security app hosts list api
```

Comment:
I'd hide implementation details for API:

```gherkin
    And the host name is shown in the security app
```

Reply:
i don't have a problem with that change, will update.  thanks Manu